examples/app_us3_voice/src/app_us3_voice.xc

```diff
@@ -4,6 +4,10 @@
 #include <stdint.h>
 #include <stdio.h>
 #include <string.h>
+
+//#define DEBUG_PRINT_ENABLE 1 // uncomment this line to enable debug printing
```

Comment:
Same comment as about re creating a redefinition of the `DEBUG_PRINT_ENABLE ` define.